tests/integration/test_reddit.py

```diff
@@ -13,6 +15,36 @@
 
 
 class TestReddit(IntegrationTest):
+    def test_bad_request_without_json_text_plain_response(self):
+        with open("tests/integration/files/too_large.jpg", "rb") as fp:
+            junk = urlsafe_b64encode(fp.read()).decode()
+        with self.use_cassette(
+            placeholders=self.recorder.configure().default_cassette_options[
```

Comment:
Will this trick work in the next test as well?

Reply:
Yes, it would work. I could also do the same thing for a very similar [test](https://github.com/praw-dev/praw/blob/master/tests/integration/models/reddit/test_wikipage.py#L38) I added recently, which would turn a [large cassette](https://github.com/praw-dev/praw/blob/master/tests/integration/cassettes/TestWikiPage.test_edit__usernotes.json) into a [much smaller one.](https://github.com/MaybeNetwork/praw/blob/ac108cd4f6c5f8a1406ac1f5ad578625eafd7a4c/tests/integration/cassettes/TestWikiPageCompare.test_edit__usernotes.json)

Reply:
Great, could you add those changes to this PR please?

Reply:
Sure. Should I rebase again?  The first rebase happened before the second docstrfmt PR went through.